src/NuGet.Core/NuGet.ProjectModel/JsonPackageSpecReader.cs

```diff
@@ -524,6 +524,128 @@ private static IEnumerable<FrameworkRuntimePair> ReadCompatibilitySets(JsonTextR
             });
         }
 
+        internal static void ReadCentralTransitveDependencyGroup(
```

Comment:
Might be a good idea to add tests for the reader + writer. 

@dtivel recently (within the last year) rewrote this and added *lots* of coverage. 

https://github.com/NuGet/NuGet.Client/blob/dev/test/NuGet.Core.Tests/NuGet.ProjectModel.Test/JsonPackageSpecReaderTests.cs

Reply:
added